src/main/java/berlin/yuna/nano/services/metric/logic/MetricService.java

```diff
@@ -66,6 +71,56 @@ public void onEvent(final Event event) {
                 metrics.gaugeSet("logger", 1, Map.of("level", level.name()));
             })
             .ifPresent(EVENT_APP_LOG_QUEUE, LogQueue.class, logger::logQueue);
+        addMetricsEndpoint(event);
+
+    }
+
+    protected void addMetricsEndpoint(Event event) {
+        event
+            .ifPresent(EVENT_HTTP_REQUEST, HttpObject.class, request ->
+            {
+                request.pathMatch(Config.getMetricsBaseUrl() + Config.getPrometheusBaseUrl());
```

Comment:
It should be `Config.getMetricsBaseUrl() + "prometheus"` or `Config.getPrometheusUrl()` if it's set.
Also please take care if the first and last `/` in `Config.getMetricsBaseUrl()` and `Config.getPrometheusUrl()`. Users could add or forget them:
* NO: metrics/prometheus
* NO: metrics/prometheus/
* NO: /metrics/prometheus/
* YES: /metrics/prometheus